api/starknet_trace_api_openrpc.json

```
@@ -37,6 +37,77 @@
                 }
             ]
         },
+        {
+            "name": "starknet_simulateTransaction",
+            "summary": "simulate a given transaction on the requested state, and generate the execution trace",
+            "params": [
+                {
+                    "name": "block_id",
+                    "description": "The hash of the requested block, or number (height) of the requested block, or a block tag, for the block referencing the state or call the transaction on.",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/BLOCK_ID"
+                    }
+                },
+                {
+                    "name": "transaction",
+                    "description": "The transaction to simulate",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/BROADCASTED_TXN"
+                    }
+                },
+                {
+                    "name": "simulation_flags",
+                    "description": "describes what parts of the transaction should be executed",
+                    "required": true,
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/components/schemas/SIMULATION_FLAG"
+                        }
+                    }
+                }
+            ],
+            "result": [
+                {
+                    "name": "simulated_transaction",
+                    "description": "The execution trace and consuemd resources of the required transaction",
+                    "schema": {
+                        "type": "object",
+                        "properties": {
+                            "transaction_trace": {
+                                "title": "the transaction's trace",
+                                "$ref": "#/components/schemas/TRANSACTION_TRACE"
+                            },
+                            "fee_estimation": {
+                                "title": "the transaction's resources and fee",
+                                "$ref": "#/components/schemas/FEE_ESTIMATE"
+                            }
+                        }
+                    }
+                }
+            ],
```

I think you need to drop the outer array too:

```suggestion
            "result": {
                "name": "simulated_transaction",
                "description": "The execution trace and consumed resources of the required transaction",
                "schema": {
                    "type": "object",
                    "properties": {
                        "transaction_trace": {
                            "title": "the transaction's trace",
                            "$ref": "#/components/schemas/TRANSACTION_TRACE"
                        },
                        "fee_estimation": {
                            "title": "the transaction's resources and fee",
                            "$ref": "#/components/schemas/FEE_ESTIMATE"
                        }
                    }
                }
            },
```